src/shared/images/SearchIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function SearchIcon() {
```

Comment:
@sniedzielski @KarolinaKopacz would it not be better to use a library with icons? You could use this one https://react-icons.github.io/react-icons/ which gathers the most common icon libs.

Reply:
@awalkowiak I installed lib 'react-lib' in that commit. https://github.com/SD-TransIT/transit_frontend/pull/21/commits/10caf9757ae6315c3ee67e8b29baf299bee03fdb. This commit also includes taking <FiSearch> icon from that lib.